docs/BigQueryTable-batchsource.md

```diff
@@ -45,6 +45,9 @@ If no value is given, all partitions up to the partition end date will be read.
 **Partition End Date**: Exclusive partition end date, specified as 'yyyy-MM-dd'. For example, '2019-01-01'. 
 If no value is given, all partitions up from the partition start date will be read.
 
+**Filter**: The WHERE clause filters out rows by evaluating each row against boolean expression,
+and discards all rows that do not return TRUE (that is, rows that return FALSE or NULL).
```

Comment:
We try to write the docs with as little technical language as possible. For example, some users might not even know what a 'WHERE' clause is. Let's change it to something like:

Filters out rows that do not match the given condition. For example, if the filter is 'age > 50 and name is not null', all output rows will have an 'age' over 50 and a value for the 'name' field.